### DISCUSSION TITLE
receiving Sysex and display things on ili9341

### DISCUSSION THREAD

**Discussion by benwadub** [tttapa/Control-Surface#598] [Q&A]
hi, I found a way to send Sysex from my daw with help of remotely Ableton script generator, do you think it is possible to receive theses Sysex and print something on my display to show my mutes on or off?
by example receiving this Sysex F0 41 10 42 12 40007F 00 41 F7 print mute1  or light a part on the screen?
i have seen the send receive example but don t know how to write the code for each specific Sysex write or delete something specific.
thanks

**Comment by tttapa** [comment]
> but don t know how to write the code for each specific Sysex write

Why not? What exactly are you struggling with? The SysEx data is just an array of bytes, you can do with it whatever you want, such as comparing the bytes to a specific sequence of bytes you're listening for.

**Reply by benwadub** [reply]
If I understand well the example receiving sysex do something for any Sysex  received, I don t know how to write by exemple : 
this specific sysex light this button on the screen, this other one light this other button…
Could you please juste show me one exemple for one sysex? Maybe on a display available for the library i ll adapt it to the ili9341

**Reply by tttapa** [reply]
See https://forum.arduino.cc/t/arduino-uno-midi-system-exclusive/896857/34 and https://forum.arduino.cc/t/problem-receiving-sysex-messages/865321/6.

For the display-specific stuff, you should look at the examples that come with the ILI9341 library you're using.

**Reply by benwadub** [reply]
thanks I ll look at it!
